src/view/com/posts/PostFeedItem.tsx

```diff
@@ -258,7 +262,7 @@ let FeedItemInner = ({
       }}>
       <SubtleWebHover hover={hover} />
       <View style={{flexDirection: 'row', gap: 10, paddingLeft: 8}}>
-        <View style={{width: 42}}>
+        <View style={{width: isCarouselItem ? 0 : 42}}>
```

Comment:
why set the width to 0 vs not render the component?

Reply:
I believe this was a fix for some layout bug and not something directly affecting visibility.